db/seeds.rb

```diff
@@ -1,7 +1,17 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+User.create(email: 'email@example', password: 'password')
+AdminUser.create(email: 'admin@example', password: 'password')
+3.times do |n|
+  category = Category.new(name: "Category#{n}")
+  brand = Brand.new(name: "Brand#{n}")
+
+  Category.where(name: category.name).first_or_create(category.as_json)
+  Brand.where(name: brand.name).first_or_create(brand.as_json)
+end
+10.times do |n|
+  product = Product.new(name: "Product#{n}",
+                        price: rand(1000.01..99999.99).round(2),
+                        brand: Brand.all.sample,
+                        category: Category.all.sample )
+                        
+  Product.where(name: product.name).first_or_create(product.as_json)
```

Comment:
Kiedy skonfigurowana będzie jakaś chmura to tutaj dodam coś w stylu
```suggestion
  product = Product.where(name: product.name).first_or_create(product.as_json)
  product.photos.attach(io: File.open('/path/to/file'), filename: 'file.jpg')
```

Reply:
Chyba, że dodać to w ifa 
```
if !Rails.env.production?
```